samples/subsys/ipc/ipc_service/icmsg_me/sysbuild.cmake

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+ExternalZephyrProject_Add(
+	APPLICATION remote
+	SOURCE_DIR  ${APP_DIR}/remote
+	BOARD       ${SB_CONFIG_NET_CORE_BOARD}
+)
```

Comment:
If that is the case then you can instead put an error here if `SB_CONFIG_NET_CORE_BOARD` is not set and means that only a mapping board list needs added to the Kconfig.sysbuild file